src/document.c

```diff
@@ -3392,6 +3392,16 @@ static void force_close_all(void)
 	main_status.closing_all = FALSE;
 }
 
+void force_reload_all(void)
```

Comment:
Since this is only called once suggest you put it in the callback, Geany is exhausting enough playing spaghetti functions :grin:

Reply:
Done